packaging/bundle-validation/validate.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# NOTE: this script runs inside hudi-ci-bundle-validation container
+# $WORKDIR/jars/ is supposed to be mounted to a host directory where bundle jars are placed
+# TODO: $JAR_COMBINATIONS should have different orders for different jars to detect class loading issues
+
+WORKDIR=/opt/bundle-validation
+HIVE_DATA=${WORKDIR}/data/hive
+JAR_DATA=${WORKDIR}/data/jars
+UTILITIES_DATA=${WORKDIR}/data/utilities
+
+run_hive_sync () {
+    echo "::warning::validate.sh setting up hive sync"
+    #put config files in correct place
+    cp $HIVE_DATA/spark-defaults.conf $SPARK_HOME/conf/
+    cp $HIVE_DATA/hive-site.xml $HIVE_HOME/conf/
+    ln -sf $HIVE_HOME/conf/hive-site.xml $SPARK_HOME/conf/hive-site.xml
+    cp $DERBY_HOME/lib/derbyclient.jar $SPARK_HOME/jars/
+
+    $DERBY_HOME/bin/startNetworkServer -h 0.0.0.0 &
+    $HIVE_HOME/bin/hiveserver2 &
+    echo "::warning::validate.sh hive setup complete. Testing"
+    $SPARK_HOME/bin/spark-shell --jars $JAR_DATA/spark.jar < $HIVE_DATA/validate.scala
+    if [ "$?" -ne 0 ]; then
+        echo "::error::validate.sh failed hive testing)"
+        exit 1
+    fi
+    echo "::warning::validate.sh hive testing succesfull. Cleaning up hive sync"
+    #remove config files
+    rm -f $SPARK_HOME/jars/derbyclient.jar
+    unlink $SPARK_HOME/conf/hive-site.xml
+    rm -f $HIVE_HOME/conf/hive-site.xml
+    rm -f $SPARK_HOME/conf/spark-defaults.conf
+}
+
+test_utilities_bundle () {
+    echo "::warning::validate.sh running deltastreamer"
+    $SPARK_HOME/bin/spark-submit --driver-memory 8g --executor-memory 8g \
+    --class org.apache.hudi.utilities.deltastreamer.HoodieDeltaStreamer \
+    $JARS_BEGIN $JARS_ARG \
+    --props $UTILITIES_DATA/parquet-dfs-compact.props \
+    --schemaprovider-class org.apache.hudi.utilities.schema.FilebasedSchemaProvider \
+    --source-class org.apache.hudi.utilities.sources.ParquetDFSSource \
```

Comment:
it is not a good practice to version control parquet files (binary data) for testing, not easy to inspect what the content is and hard to evolve. we should use JsonDFSSource and make use of the stocks data from docker/demo/data/. cc @nsivabalan thought we agreed on this ?